tests/unit/test_coupling.py

```diff
@@ -83,20 +92,70 @@ def test_real_output():
 
 @pytest.mark.basic
 def test_closest_tune_approach():
+    desire = namedtuple('desire', ['err', 'isreal'])
+    map = {'teapot': desire(0, True),  # results are compared to this, from a madx-match would maybe better
+           'calaga': desire(0, True),  # same method as teapot
+           'franchi': desire(0.001, True),
+           'teapot_franchi': desire(0.0005, True),
+           'persson': desire(0.25, False),  # Not sure why it is so high here
+           'persson_alt': desire(0.25, False),
+           'hoydalsvik': desire(0.25, False),
+           'hoydalsvik_alt': desire(0.25, False),
+           }
+
```

Comment:
This feels quite convoluted.

Reply:
`pytest.mark.parametrize`?

Reply:
but then I need to calculate the cta twice, once for the method and once for `teapot` to be able to compare it with this. Also I need to get df_twiss and df_cmatrix. maybe put it in a fixture which also outputs the result for teapot... I don't think it becomes more readable

Reply:
I did.